src/printer_xml.c

```diff
@@ -54,19 +55,30 @@ modlist_add(struct mlist **mlist, const struct lys_module *mod)
         LY_CHECK_ERR_RETURN(!iter, LOGMEM(mod->ctx), EXIT_FAILURE);
         iter->next = *mlist;
         iter->module = (struct lys_module *)mod;
-        *mlist = iter;
+	iter->printed = 0;
+	*mlist = iter;
```

Comment:
Still a TAB  not spaces for indent.
And some other line for the same issue.

Reply:
Sorry about that.  I did double check for tabs.  should be fixed now.